scripts/launch-s0-instance.sh

```diff
@@ -237,27 +237,6 @@ if ! which aws >/dev/null 2>&1 || [[ $(aws --version | cut -d' ' -f1 | cut -d'/'
     exit 1
 fi
 
-# Try to install 'jq' if it's not existing or fail
-if ! which jq; then
```

Comment:
@e-kov are tr and awk installed by default on all Linux distros? if yes then it's cool) if not then we still will get errors like tr: command not found or awk: command not found

Reply:
yes, they are always present even in the minimal installs of the supported distros.